packages/web/src/scss/components/UNSTABLE_ProductLogo/index.html

```diff
@@ -0,0 +1,99 @@
+{{#> web/layout/plain }}
+
+<section class="docs-Section">
+  <h2 class="docs-Heading">Default</h2>
+  <div class="docs-Stack docs-Stack--stretch">
```

Comment:
There is only one item in this `Stack` component – no need to use it. (same for line 40)